orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/model/GoogleCloudBuildStageDefinition.java

```diff
@@ -20,22 +20,26 @@
 import lombok.Getter;
 
 import java.util.Map;
+import java.util.Optional;
 
 @Getter
-public class GoogleCloudBuildStageDefinition {
+public class GoogleCloudBuildStageDefinition implements RetryableStageDefinition {
   private final String account;
   private final GoogleCloudBuild buildInfo;
   private final Map<String, Object> buildDefinition;
+  private final int consecutiveErrors;
 
   // There does not seem to be a way to auto-generate a constructor using our current version of Lombok (1.16.20) that
   // Jackson can use to deserialize.
   public GoogleCloudBuildStageDefinition(
     @JsonProperty("account") String account,
     @JsonProperty("buildInfo") GoogleCloudBuild build,
-    @JsonProperty("buildDefinition") Map<String, Object> buildDefinition
+    @JsonProperty("buildDefinition") Map<String, Object> buildDefinition,
+    @JsonProperty("consecutiveErrors") Integer consecutiveErrors
```

Comment:
Is there a particular reason this `consecutiveErrors` is `Integer` vs `int`?

Reply:
It's `Integer` in the constructor as it might be null if it's unset in the JSON; we convert to a primitive `int` before setting it on the object so consumers can just use a `int`.